src/components/FromSTIX/utils.js

```diff
@@ -66,3 +68,12 @@ export function filterMappingFieldsForValue(mappings, value) {
       return obj;
     }, {});
 }
+
+export function updateVersionOfStix(num){
+  let version = stixLangV2;
+  if(num===1){
+    let version = stixLangV2_1;
```

Comment:
version is declared twice which is incorrect, 
moreover, can we generalize it if tomorrow we would have another version? 
why do you have to import `stixLangV2` on both files `utils.js` and `ChangeVersion.jsx`?